ammo-be/src/routers/bulletRouter/__tests__/ws.test.ts

```diff
@@ -78,4 +79,61 @@ describe('[WS] BulletRouter', () => {
             done();
         });
     });
+
+    it('The recording state must be initialized to true', () => {
+        expect(isRecording).toBe(true);
```

Comment:
Where is 'isRecording' coming from?

Reply:
This is the same boolean that allow the server to send bullet to front or not